qa/tasks/cephfs/test_mds_metrics.py

```diff
@@ -450,3 +471,48 @@ def test_perf_stats_stale_metrics(self):
         finally:
             # cleanup test directories
             self._cleanup_test_dirs()
+
+    def test_client_metrics_and_metadata(self):
+        self.mount_a.umount_wait()
+        self.mount_b.umount_wait()
+
+        self.mds_cluster.mon_manager.raw_cluster_cmd("fs", "flag", "set",
+            "enable_multiple", "true",
+            "--yes-i-really-mean-it")
+            
+        #creating two filesystems
+        fs_a, fs_b = self._setup_two_fs()
```

Comment:
Can we have more generic function and call it twice?
Eg: setup_fs(name)

Reply:
Yeah, we can have it.Actually, I created for this test case thats why. Okay, generic one will be better.